src/fty_common_db_asset.cc

```diff
@@ -679,7 +748,7 @@ int select_assets_all_container(tntdb::Connection& conn, std::vector<uint16_t> t
             " FROM "
             "   t_bios_asset_element as t";
 
-        if (!subtypes.empty() || !types.empty() || status != "" || without != "") {
+        if (!subtypes.empty() || !types.empty() || status != "" || without != "" || !configured.empty()) {
```

Comment:
if configured is non empty and not "yes" or "no"... the select request is malformed (the WHERE part is not set)?